.github/workflows/release.yaml

```
@@ -11,48 +8,77 @@ on:
-          - os: macos-latest
+          - build: macos
+            os: macos-latest
             target: aarch64-apple-darwin
-            binary_name: app-darwin-arm64
-    runs-on: ${{ matrix.os }}
+
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)